### PR TITLE
docs(changelog): add doc for rxjs multicasting

### PR DESCRIPTION
**Pull request by subratpalhar92**
add documentation demonstrating rxjs multicasting and show a pagination use case with rxjs subject and give an example

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by subratpalhar92** [comment]
This was an earlier pull request

https://github.com/angular/angular/pull/37024

File: aio/content/guide/observables.md [hunk 1]
```diff
@@ -101,9 +101,13 @@ Notice that if you subscribe twice, there will be two separate streams, each emi
 
 <code-example path="observables/src/multicasting.ts" region="multicast_sequence" header="Create a multicast subscriber"></code-example>
 
-<div class="alert is-helpful">
-   Multicasting observables take a bit more setup, but they can be useful for certain applications. Later we will look at tools that simplify the process of multicasting, allowing you to take any observable and make it multicasting.
-</div>
+### Multicasting with subject
```
Comment (kapunahelewong) [inline aio/content/guide/observables.md:104]:
```suggestion
### Multicasting with a subject
```

File: aio/content/guide/observables.md [hunk 2]
```diff
@@ -101,9 +101,13 @@ Notice that if you subscribe twice, there will be two separate streams, each emi
...
+
+In RxJS, a Subject is a special type of Observable that allows values to be multicasted to many Observers.
```
Comment (kapunahelewong) [inline aio/content/guide/observables.md:106]:
```suggestion
In RxJS, a `subject` is a special type of observable that allows values to be multicast to many Observers.
```

File: aio/content/guide/observables.md [hunk 3]
```diff
@@ -101,9 +101,13 @@ Notice that if you subscribe twice, there will be two separate streams, each emi
...
+
+In the following example, several observers receive the same current-page value emitted by the Subject. Each one handles the received value differently to log the current-page value, the next-page value, and the previous-page value.
```
Comment (kapunahelewong) [inline aio/content/guide/observables.md:108]:
```suggestion
In the following example, the `subject` emits the same current-page value to several observers. 
Each observer handles the received value differently when logging the current-page value, the next-page value, and the previous-page value.
```

**Review by kapunahelewong — COMMENTED** [review]
Thanks for this thoughtful update, @subratpalhar92! I've left a few small suggestions. We will still need engineering review and approval. If you have any questions throughout the process just let me know!

**Comment by kapunahelewong** [comment]
Thank you for making the updates, @subratpalhar92. I really like that you have included an example. Could you explain a little bit more about the example that you have included? For example, you may want to describe what the `pagination` `const` is and what each one of those `subscribe` functions do. You could do this in two or three sentences after the example code.

We do still need engineering approval about the actual content, but you may want to keep these points in mind too as we iterate. Also at some point you will want to rebase as that will help you pass CircleCI. Here are [my notes](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-rebase) on rebasing if you would find them helpful.

**Comment by AndrewKushnir** [comment]
Closing this PR since there was no reply from PR author. Please reopen or create a new PR when you get a chance to look into that again. Thank you.

**Pull request closed without merging**